### ISSUE TITLE
Descriptions of hybrid pruning 

### ISSUE THREAD

**Issue by xiamengzhou** [huggingface/nn_pruning#9]
I am wondering where I can find descriptions of the hybrid pruning method as it is not covered in the original movement pruning paper? Thanks so much!

**Comment by madlag** [comment]
Yes, it's described in the upcoming EMNLP 21 paper, available on arxiv too: https://arxiv.org/abs/2109.04838 .
